bin/cut.py

```diff
@@ -2,7 +2,6 @@
 """Print selected parts of lines from each FILE to standard output."""
 
 from __future__ import print_function
-import os
```

Comment:
Please make a separate PR to make just this one change.

Why?  Because it is safe, and once you have a PR merged, then GitHub Actions will automatically run for you.

Reply:
@o-murphy